src/rupali/Assignment8.java

```diff
@@ -0,0 +1,69 @@
+package rupali;
+
+public class Assignment8 {
+	
+	void isEven(int evencount) {
```

Comment:
we need to identify number with given range 

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files